src/chrome/content/rules/Freecycle.xml

```diff
@@ -4,10 +4,13 @@
 	<target host="butler.freecycle.org"/>
 	<target host="mail.freecycle.org"/>
 	<target host="webmail.freecycle.org"/>
+	<target host="my.freecycle.org"/>
+	<target host="www.freecycle.org"/>
+	<target host="freecycle.org"/>
+	<target host="groups.freecycle.org"/>
 
 	<securecookie host="^(?:web)?mail\.freecycle\.org$" name=".*"/>
 
-	<rule from="^http:"
-		to="https:" />
+	<rule from="^http:"	to="https:" />
```

Comment:
No, no particular reason. I just saw that 'make-trivial-rule' does it like that. The main part are the additional subdomains here.